src/components/Todo.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from "react";
+import { AiFillDelete } from "react-icons/ai";
+import { FaEdit } from "react-icons/fa";
+import { ImCross } from "react-icons/im";
+
+export const Todo = () => {
+  const [text, setText] = useState("");
+  const [todos, setTodos] = useState([]);
+  const [editId, setEditID] = useState(0);
+  const [openTodo, setOpenTodo] = useState(false);
+
+ const handleSubmit=(e)=>{
+    e.preventDefault();
+    if (editId) {
+      const editTodo = todos.find((i) => i.id === editId);
+      const updateTodo = todos.map((item) =>
+        item.id === editTodo.id
+          ? (item = { id: item.id, text })
+          : { id: item.id, text: item.text, completed: false }
```

Comment:
Appreciation: Everything is looking great. The code is readable and clean.
Suggestion: You can destructure the 'item' and assign values.

Reply:
Ok